src/main/java/ch/njol/skript/effects/EffBroadcast.java

```diff
@@ -76,33 +70,17 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	}
 	
 	@Override
-	@SuppressWarnings("deprecation")
-	public void execute(Event e) {
+	public void execute(Event event) {
 		List<CommandSender> receivers = new ArrayList<>();
```

Comment:
I think a linked list might be more efficient to add to, or you can give the array an initial capacity since you know the size of [the receivers] at this point.